### ISSUE TITLE
TODO: integrate exomeQualityPlots scripts into LG3 pipeline

### ISSUE THREAD

**Issue by ivan108** [UCSF-Costello-Lab/LG3_Pipeline#88]
Stephanie created a pipeline to generate quality plots on top of the results of LG3 pipeline, see
[exomeQualityPlots](https://github.com/SRHilz/exomeQualityPlots).

According to current procedure, I ran this pipeline after LG3 is done. I think it makes sense to include  exomeQualityPlots scripts into LG3 pipeline.

**Comment by HenrikBengtsson** [comment]
Awesome.  What steps in the LG3 Pipeline needs to be completed before exomeQualityPlots can be run?

/cc @SRHilz

**Comment by ivan108** [comment]
I support alternative #1.
In fact I don't use the original exomeQualityPlots pipeline as is, but developed a more comfortable interface to it (launching scripts). It is like a branch from the original pipeline, but no version control. Now I adapted interface to our coding style and variable names, so it becomes just another steps of LG3.
The pipeline addition has 3 new steps:
```
_run_QC_1
_run_QC_2
_run_QC_3
```
_run_QC_1 and _run_QC_2 can be run in parallel, and _run_QC_3 after 1 and 2 are done..

With the second alternatives we would have to first substantially refactor original exomeQualityPlots pipeline and then to write some loading modules...

**Comment by HenrikBengtsson** [comment]
Hmm... you closed the issue.  If you want to go with Alt 1, then you need to have @SRHilz onboard.

> In fact I don't use the original exomeQualityPlots pipeline as is, but developed a more comfortable interface to it (launching scripts).

Unless there are major changes, you should be able to write your own wrapper scripts that achieve what you've done but that just call the original exomeQualityPlots pipeline with the proper setup.

> ... It is like a branch from the original pipeline, but no version control. 

Sorry, but nope. It's not.  You've forked, or rather copied the latest version, her work and now there are two code bases that have to be maintained.  If this would have been a small script, it might be acceptable, but there's lots of code in there. You don't want to do this.  "Trust me".  Maintenance of software is hard enough.  This "forking" adds work for @SRHilz - if not today, it'll get to her at some point in the future.

**Comment by ivan108** [comment]
I totally agree that we don't want to version control the same code in two places.
The simple solution could be to clone exomeQualityPlots pipeline elsewhere and make symbolic links from our scripts/ directory. The only small complication is when we want to fix a bug, we would have to go through Stephanie. 
BTW, I got consent from Stephanie to use her scripts in LG3 pipeline first thing...

**Comment by HenrikBengtsson** [comment]
> I am fine with whatever is easiest. Ideally it would be great to use a clone of my repository, just to keep things tidy. Can I set you both up as contributors on that repo as well, so that you can fix bugs on it as well?

Thanks. Yes, working toward a single code base is key (here and in all other projects).  Giving us direct push writes to your repository is the most straightforward and probably entails fewer surprises/hiccups.  The alternative is that we work toward a (git) **fork** on your repository and when we think we have something that's worth incorporating, we send you a pull request that you accept/merge.

**Comment by HenrikBengtsson** [comment]
@ivan108, could you please drop those new files again? `make check` fails because of them.
